firecares/tasks/weather_task.py

```diff
@@ -9,3 +10,25 @@ def collect_weather_noaa_warnings():
     This harvest can take up to 10 minutes to iterate through all warnings
     """
     WeatherWarnings.load_warning_data()
+
+
+@app.task(queue='weather-task', rate_limit='24/h')
+def cleanup_dept_weather_noaa_warnings():
```

Comment:
The commit that has this change can be killed from this branch.  Actually, be sure that any of the weather-related commits are not included in this PR.  If they need to be pulled-out and moved to a separate PR, let me know, but I did just merge in the latest weather-related cleanup item.